diagnostics-app/src/manifest.webmanifest

```diff
@@ -1,9 +0,0 @@
-{
```

Comment:
I vaguely remember this file is used for something else, likely for PWA. Can you verify whether deleting this file will still allow for compilation for the normal PWA workflow?

Reply:
Thanks for the feedback. Let's keep this file for now.